### ISSUE TITLE
Tests should pass by running `pytest`

### ISSUE THREAD

**Issue by K0Te** [K0Te/django-object-manager#4]
For some reason tests pass only when running `setup.py test`, but running just `pytest` fails, example:
```
$ pytest
usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --cov=django_object_manager
  inifile: /Users/kote/github/django-object-manager/setup.cfg
  rootdir: /Users/kote/github/django-object-manager
```

`setup.py test` uses `pytest` under the hood, so it should be simple to use `pytest` directly

**Comment by dnafication** [comment]
Hi @K0Te, I was trying to set it up for the first time, seems like running `setup.py install` doesn't install test requirements. How should I get test requirements installed?

**Comment by Tadaboody** [comment]
I think it's because `setup.py` specifies that the tests are in `tests/functional` https://github.com/K0Te/django-object-manager/blob/e0b55877d0459a481e32419e5ad33758d10d5dc5/setup.py#L32
Pytest defaults that tests are in `tests`. Running `pytests tests/functional` should work

**Comment by K0Te** [comment]
Hi @dnafication 
`python3.6 setup.py test` automatically installs tests requirements.
It might be a good idea to create `requirements.txt` with both package and test requirements. PRs are welcome !

After re-checking this issue with fresh clear venv everything works fine:
```
python3.6 -m venv venv
. venv/bin/activate
pip install pytest-django pytest pytest-cov
$ pytest 
=============================================================================== test session starts ===============================================================================
platform darwin -- Python 3.6.4, pytest-3.8.2, py-1.6.0, pluggy-0.7.1 -- /Users/kote/github/django-object-manager/venv/bin/python3.6
cachedir: .pytest_cache
rootdir: /Users/kote/github/django-object-manager, inifile: setup.cfg
plugins: django-3.4.3, cov-2.6.0
collected 9 items                                                                                                                                                                 

tests/functional/test_object_manager.py::TestPlaneMake::test_double_get PASSED                                                                                              [ 11%]
tests/functional/test_object_manager.py::TestPlaneMake::test_foreign_key_predefined PASSED                                                                                  [ 22%]
tests/functional/test_object_manager.py::TestPlaneMake::test_foreing_key_as_instance PASSED                                                                                 [ 33%]
tests/functional/test_object_manager.py::TestPlaneMake::test_many_to_many_forward_predefined PASSED                                                                         [ 44%]
tests/functional/test_object_manager.py::TestPlaneMake::test_many_to_many_reversed_predefined PASSED                                                                        [ 55%]
tests/functional/test_object_manager.py::TestPlaneMake::test_multiple_predefined PASSED                                                                                     [ 66%]
tests/functional/test_object_manager.py::TestPlaneMake::test_namespace PASSED                                                                                               [ 77%]
tests/functional/test_object_manager.py::TestPlaneMake::test_single_from_kwargs PASSED                                                                                      [ 88%]
tests/functional/test_object_manager.py::TestPlaneMake::test_single_predefined PASSED                                                                                       [100%]

---------- coverage: platform darwin, python 3.6.4-final-0 -----------
Name                                      Stmts   Miss  Cover
-------------------------------------------------------------
django_object_manager/__init__.py             2      0   100%
django_object_manager/object_manager.py     123     17    86%
-------------------------------------------------------------
TOTAL                                       125     17    86%


============================================================================ 9 passed in 0.56 seconds =============================================================================
```

@Tadaboody it seems that `test_suite` might be removed from setup.py, as pytest does nice test discovery.

**Comment by dnafication** [comment]
Thank you for clarifying that. I'll work on adding the requirements.txt 😛

**Comment by K0Te** [comment]
Closed by #7
